exercises/strain/StrainExample.m

```diff
@@ -0,0 +1,31 @@
+#import <Foundation/Foundation.h>
+#import "StrainExample.h"
+
+@implementation NSArray (StrainExample)
+
+-(instancetype)keep:(BOOL)compare where:(BOOL (^)(id _Nonnull arrayElement))completionHandler{
+    
+    NSMutableArray *filteredArray = [NSMutableArray array];
+    
+    [self enumerateObjectsUsingBlock:^(id  _Nonnull objectInArray, NSUInteger idx, BOOL * _Nonnull stop) {
```

Comment:
I’d recommend changing this to a simple for - in loop, since the index and stop parameters aren’t needed here.